Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -1214,11 +1212,12 @@ void MediaPlayerPrivateGStreamer::notifyPlayerOfTrack()
         return;
 
     unsigned numberOfTracks = 0;
-    StringPrintStream numberOfTracksProperty;
-    numberOfTracksProperty.printf("n-%s", typeName);
-    g_object_get(m_pipeline.get(), numberOfTracksProperty.toCString().data(), &numberOfTracks, nullptr);
+    StringBuilder builder;
+    builder.append("n-"_s, typeName);
+    auto numberOfTracksProperty = builder.toString();
```

Comment:
Any reason to not use `makeString()`? I think the following would work:


```suggestion
    auto numberOfTracksProperty = makeString("n-"_s, typeName);
```